test/test_plugins.py

```diff
@@ -4,6 +4,7 @@
 import sys
 import unittest
 from pathlib import Path
+from yt_dlp import Config
```

Comment:
should this be moved below where we import from `yt_dlp.plugins`?

Reply:
Done